packages/react-components/src/molecules/MembersList.tsx

```diff
@@ -104,10 +109,10 @@ const MembersList: React.FC<MembersListProps> = ({
               css={[
                 addToColumnStyles,
                 singleColumn || multiColumnAddToColumnStyles,
-                roleStyles,
+                labStyles,
```

Comment:
we can keep the `roleStyles`and add the `labStyles´ with the padding so we don't set the color in two different places

Reply:
In that case, we should change the name to something more generic. I'm using `textStyles`